en/examples/README.md

```diff
@@ -1,5 +1,7 @@
 # Examples
 
+> **Tip** Information about *writing* example code is covered in the [Contributing > Writing Plugins](../contributing/plugins.md) (*plugin developers* should initially create [integration tests](../contributing/test.md#integration-tests) rather than examples for new code).

```

Comment:
@julianoes This was my first effort to make people aware that they should not write and example before integration tests. The problem with this is that most users just want to see the examples, so it feels "too prominent". Might not be necessary now we have plugin guide

Reply:
Ok that's good!